src/components/CustomInputs/DictInput.vue

```diff
@@ -224,6 +243,7 @@ export default {
                   ? dict[i].value.toString()
                   : valueLabel
               "
+              :readonly="!includedKeys.includes(keys[i])"
```

Comment:
This should only be disabled if the `includeCheckbox` prop is passed as well, right?

Reply:
oooh my bad. I will add that now